donation-api/src/main/java/org/interfaithsanctuary/donationapi/model/Callout.java

```diff
@@ -9,35 +9,29 @@
     @Id
     @Column(name = "callout_id")
     @GeneratedValue(strategy = GenerationType.AUTO)
-    private Long calloutId;
+    private Long id;
 
-    @Column(name = "callout_title")
+    @Column(name = "callout_name")
     private String title;
 
-    @Column(name = "callout_body")
-    private String body;
+    @Column(name = "callout_effective_date")
+    private Date effectiveDate;
 
-    @Column(name = "callout_url")
-    private String url;
+    @Column(name = "callout_active")
+    private boolean active;
 
-    @Column(name = "create_date")
-    private Date createDate;
+    @Column(name = "callout_date_created")
+    private Date createdDate;
 
-    @Column(name = "update_date")
-    private Date updateDate;
+    @Column(name = "callout_description_message")
+    private String descriptionMessage;
 
-    @Column()
-    private Boolean active;
-
-    @Column()
-    private Boolean pinned;
-
-    public Long getCalloutId() {
-        return calloutId;
+    public Long getId() {
+        return id;
     }
 
-    public void setCalloutId(Long calloutId) {
-        this.calloutId = calloutId;
+    public void setId(Long id) {
+        this.id = id;
     }
 
     public String getTitle() {
```

Comment:
same as other comment in code review, I'm fine with it being called title but it might be confusing when querying from database